src/components/map.js

```diff
@@ -188,6 +189,26 @@ export class SkraaFotoMap extends HTMLElement {
         })
       })
 
+      if (configuration.ENABLE_POINTER) {
+        map.addLayer(generatePointerLayer())
+        map.on('pointermove', event => {
+          /**
+           * It is too expensive to get the Z value for every point, so we use the most recent value stored from
+           * zoom sync instead.
+           */ 
+          const coord = [event.coordinate[0], event.coordinate[1], store.state.view.center[2] || 0]
+          window.dispatchEvent(new CustomEvent("updatePointer", { detail: { coord: coord, map: map } }))
+        })
+        window.addEventListener('updatePointer', event => {
+          if (event.detail.map === map) {
+            updatePointer(this.map, [-9999, -9999])
+          } else {
+            const coord = event.detail.coord
+            updatePointer(map, [coord[0], coord[1]])
+          }
+        })
+      }
```

Comment:
Hvis koden inde i denne `if` kan laves en funktion for sig og hentes ind via et modul, så kan esbuild chunke JS, således at man ikke loader den del i browseren, hvis `ENABLE_POINTER === false`